src/testswarm-browserstack.js

```diff
@@ -329,7 +329,10 @@ self = {
 				}
 				// Some browserstack workers don't define browser_version.
 				if ( bswDesc.browser_version ) {
-					parts = bswDesc.browser_version.split( '.' );
+					versionMatch = bswDesc.browser_version.match( /([\d.]+)( [a-zA-Z]+)?/ );
```

Comment:
Could you add a comment here with a sample of a real value that this will be matched against? I'm guessing this is going to contain something like "1.2.3 beta" with the implicit agreement that BrowserStack only add `<space><word>` to to describe non-stable versions?

Reply:
Exactly. I've just added a comment.